R/mod_marg.R

```diff
@@ -52,6 +64,20 @@ mod_marg2 <- function(mod, var_interest,
     all(names(at) %in% names(data))
   )
 
+  if(is.null(dof) & !is.null(vcov_mat) & mod$family$family == 'gaussian')
+    warning(paste(
+      "You provided a new variance-covariance matrix for an OLS model",
```

Comment:
I'm not sure whether this belongs as a warning or as a note under `Details`. I'm just not familiar enough with DOF corrections.